scripts/test-infra/test-e2e-presubmit.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/sh
+set -o errexit
+
+# Configure environment
+export IMAGE_REGISTRY="localhost:5001"
+export IMAGE_TAG_NAME=$(git describe --tags --dirty --always)
```

Comment:
Should be enough to customize registry. All others could be defaults

```suggestion
export IMAGE_REGISTRY="localhost:5001"
```

Reply:
Ah right

Reply:
actually `IMAGE_TAG_NAME` is still needed though because we use the unique commit hash for naming the cluster, which happens before we run Makefile where the default value is given.

Reply:
OK. Maybe the variable could be then renamed to reflect that? To `CLUSTER_NAME`, `GIT_COMMIT_NAME` or smth

Reply:
Sure, fixed.